perception/object_recognition/prediction/map_based_prediction/src/map_based_prediction_ros.cpp

```diff
@@ -47,8 +41,18 @@
 #include "map_based_prediction.h"
 #include "map_based_prediction_ros.h"
 
+std::string toHexString(const unique_identifier_msgs::msg::UUID & id){
+  auto begin = id.uuid.begin();
+  auto end = id.uuid.end();
+  std::ostringstream hex;
+  hex << std::hex;
```

Comment:
This will not pad integers with a 0 to ensure that their length is 2. See https://github.com/tier4/Pilot.Auto/blob/ros2/perception/util/visualizer/dynamic_object_visualization/include/dynamic_object_visualization/dynamic_object_visualizer.h#L66-L70 for example how I re-implemented `toHexString`, perhaps this should go into the `unique_identifier_msgs` package

Reply:
@esteve if there are already 2 nodes sharing the functionality it definitively seems like it should be common, I've seen also in [autoware_perception_rviz_plugins](https://github.com/tier4/Pilot.Auto/blob/master/common/util/rviz_plugins/autoware_perception_rviz_plugin/src/tools/car_pose.cpp#L137) that there are more UUID-related functions that seem to be commonly used

Reply:
@esteve  I've updated the function in the latest commit